aiohttp/web.py

```diff
@@ -426,6 +426,51 @@ def _make_server_creators(handler, *, loop, ssl_context,
     return server_creations, uris
 
 
+async def run_async_app(app, *, host=None, port=None, path=None, sock=None,
```

Comment:
This code very similar what we have in `run_app`, should we somehow reuse this logic in `run_app`, example is not correct but still:
```python
def run_app(...):
    loop.run_until_complet(run_async_app(...))
```